### DISCUSSION TITLE
Meaning of calculating edge attributes

### DISCUSSION THREAD

**Discussion by hkim716** [pyg-team/pytorch_geometric#4003] [Q&A]
Hi Matt, 

In the ```Cartesian transform```, we calculate ```edge_attr``` from ```edge_index``` and ```pos```. 

In the script of ```Cartesian transform```, 

``` python
cart = pos[row] - pos[col]
max_value = cart.abs().max()
cart = cart / (2 * max_value) + 0.5
```

I guess ```cart``` means the relative distance between the two connected nodes, and ```max_value``` means the distance where two node has the biggest ```cart``` value. 
Then what is the meaning of the final ```cart``` value, which will become ```edge_attr```?

**Comment by rusty1s** [comment]
You are right. The final `cart` value is thus normalized to interval `[0, 1]`.
